xml/admin_saltcluster.xml

```diff
@@ -254,21 +254,51 @@ ses-min2   /dev/vdc  hdd  8192M  450575 True
      <option>--dry-run</option> option.
     </para>
    </tip>
-   <important>
+   <sect3>
+    <title>Unmanaged OSDs</title>
     <para>
      All available clean disk devices that match the &drvgrps; specification
-     will be used as OSDs automatically after you add them to the cluster. If
-     you need to avoid this behavior, add the <option>--unmanaged=true</option>
-     option when creating OSDs:
+     will be used as OSDs automatically after you add them to the cluster. This
+     behavior is called a <emphasis>managed</emphasis> mode.  There are several
+     ways to avoid using new prepared disks automatically:
     </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       To disable the <emphasis>managed</emphasis> mode for all drives that
```

Comment:
We should advise right from the get-go to add the `unmanaged` parameter to the yaml file instead of fiddling with the cli cmd